features/project/source/browse_files.feature

```diff
@@ -133,6 +143,16 @@ Feature: Project Source Browse Files
     Then I am redirected to the files URL
     And I don't see the ".gitignore"
 
+  @javascript
+  Scenario: I can replace file and commit
+    Given I click on ".gitignore" file in repo
+    And I see the ".gitignore"
+    And I click on "Replace"
+    When I replace it with "LICENSE"
+    And I fill the commit message
+    And I click on "Replace file"
+    Then I am redirected to the ".gitignore"
+
```

Comment:
@stanhu @tsigo I also thought about that before, however, I did not figure out how to actually pick one file. Because the feature requires pick one file after click on button "choose file", then the file content can be checked. Do you have any idea? thanks.